code/lib/cli/src/sandbox-templates.ts

```diff
@@ -360,7 +360,7 @@ const baseTemplates = {
   'vue-cli/vue2-default-js': {
     name: 'Vue-CLI (Vue2 JS)',
     script:
-      'npx -p @vue/cli vue create . --default --packageManager=yarn --force --merge --preset="Default (Vue 2)" && echo "module.exports = {}" > webpack.config.js',
+      'npx -p @vue/cli vue create . --default --packageManager=yarn --force --merge --preset="Default (Vue 2)" && cd {{beforeDir}} && echo "module.exports = {}" > webpack.config.js',
```

Comment:
All these vite commands were not using {{beforeDir}} but rather `.` so it worked without this change. I think it's better to use {{beforeDir}} so can you change the `.` to {{beforeDir}} and test it if all works well?